source/documentation/deploying-an-app/using-circleci-for-continuous-deployment.html.md.erb

```diff
@@ -0,0 +1,262 @@
+          kubectl config set-credentials circleci --token=${K8S_TOKEN}
+          kubectl config set-context ${K8S_CLUSTER_NAME} --cluster=${K8S_CLUSTER_NAME} --user=circleci --namespace=${K8S_NAMESPACE}
+          kubectl config use-context ${K8S_CLUSTER_NAME}
+    - run:
+        name: Install demo helm chart
+        command: |
+          if [ "${CIRCLE_BRANCH}" == "master" ]; th en
+            helm upgrade demo helm_deploy --install --namespace=${K8S_NAMESPACE}
+          fi 
+          kubectl --namespace=${K8S_NAMESPACE} get pods
+```
+
+In the above config.yml replace `<CONTEXT>` with the name of the context you created. 
+Also replace `<REPO-NAME>` with the name of your ECR repo excluding the tag name. For example if my repo name is myteam/myrepo and my image tag is rails then I would replace
+`<REPO-NAME>` with myteam/myrepo:rails with the full repo URL being: 754256621582.dkr.ecr.eu-west-2.amazonaws.com/myteam/myrepo:rails
+
+The above config.yml needs to be saved in a folder called .circleci in the root of your application repository. 
+For demo purposes you may use the [multi-demo container repository] (https://user-guide.cloud-platform.service.justice.gov.uk/documentation/deploying-an-app/multicontainer-app-deploy.html)
+
+In the 'jobs' section we have a 'docker' executor where we will execute the steps inside the specified docker image. In this case we will run our pipeline steps inside a circle ci docker image.
+
+We then have 3 key steps as follows:
+
+1) Build / Push our application docker image. 
+
+   The multi-container demo repo has a folder called 'rails-app'. Inside this folder is a Dockerfile that defines the rail app. We run 'docker build..'
+   inside this directory. The next command is to push the image into your ECR 
+
+2) Authenticate with cluster
+
+In this step we issue a series of 'kubectl' commands to authenticate against the cluster, scoped to the namespace to which the service account belongs. 
+Earlier above we added the ${K8S_CLUSTER_CERT} and ${K8S_TOKEN} variables into our context in the CircleCI console. These variables are injected into the CircleCI image.
+The kubectl commands can then use these env variables to authenticate against the cluster. 
+
+
+3) Run Helm Chart 
+
+In the multi-container repo there is folder that contains the helm chart for the rails demo app. This step will run the helm charts and deploy the frontend.
+
+### Triggering the Pipeline
+
+The pipeline will automatically be triggered when a change to the 'master' is commmited. CircleCI will execute the latest config.yml and execute each step in the pipeline.
+
+Once all the above is complete make a change to the rails app. For example from the multi-container app repo you can amend the index html file located at 
+'/rails-app/app/views/home/index.html.erb' by changing the title and commit to master. You should now see a new job creaed in CircleCI 
```

Comment:
can you surround the filename with either single or triple backticks, to make it easier to read?

Also, "creaed" -> "created"

Reply:
done.